dockers/broad/dragmap/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM centos:centos7
+
+ARG DRAGMAP_VERSION=1.2.1 \
+        PICARD_VERSION=2.26.4 \
+        SAMTOOLS_VERSION=1.11
+
+ENV TERM=xterm-256color \
+        HAS_GTEST=0 \
+        TINI_VERSION=v0.19.0 \
+        BOOST_LIBRARYDIR=/usr/lib64/boost169 \
+        BOOST_INCLUDEDIR=/usr/include/boost169 \
+        DRAGMAP_URL=https://github.com/Illumina/DRAGMAP/archive/refs/tags/${DRAGMAP_VERSION}.tar.gz \
+        PICARD_URL=https://github.com/broadinstitute/picard/releases/download/${PICARD_VERSION}/picard.jar \
+        SAMTOOLS_URL=https://github.com/samtools/samtools/releases/download/${SAMTOOLS_VERSION}/samtools-${SAMTOOLS_VERSION}.tar.bz2
+
+LABEL MAINTAINER="Broad Institute DSDE <dsde-engineering@broadinstitute.org>"
+
+WORKDIR /usr/gitc
+
+# Install dependencies
+RUN set -eux; \
+        yum update -y; \
+        yum install -y epel-release; \
+        yum install -y boost169-devel \
+            bzip2 \
+            bzip2-devel \
+            curl-devel \
+            gcc \
+            gcc-c++ \
+            java-1.8.0-openjdk \
+            make \
+            ncurses-devel \
+            openssl-devel \
+            wget \
+            xz-devel \
+            zlib-devel \
+        ; \
+        yum clean all; \
+        rm -rf /var/cache/yum \
+        ; \
+        \
+# Install TINI
+    mkdir temp; \
+    cd temp; \
+    wget https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini -O /sbin/tini; \
+    chmod +x /sbin/tini \
+    ; \
+    \
+# Install DRAGMAP
+    wget ${DRAGMAP_URL}; \
+    tar -xf ${DRAGMAP_VERSION}.tar.gz; \
+    rm ${DRAGMAP_VERSION}.tar.gz; \
+    cd DRAGMAP-${DRAGMAP_VERSION}; \
+    make; \
+    mv build/release/dragen-os /usr/local/bin/; \
+    cd ../; \
+    rm -r DRAGMAP-${DRAGMAP_VERSION} \
+    ; \
+    \
```

Comment:
```suggestion
    wget ${DRAGMAP_URL}; \
    tar -xf ${DRAGMAP_VERSION}.tar.gz; \
    rm ${DRAGMAP_VERSION}.tar.gz; \
    \
    cd DRAGMAP-${DRAGMAP_VERSION}; \
    make; \
    mv build/release/dragen-os /usr/local/bin/; \
    \
    cd ../; \
    rm -r DRAGMAP-${DRAGMAP_VERSION} \
    ; \
```

Reply:
its good to logically separate each substep within a specific task with a line break i.e. downloading the tar, building the binary, cleaning up. Its a bit pedantic but its the style that we've chosen so its good to be consistent and makes it a little more readable